core-galleon-pack/src/main/resources/feature_groups/access-control.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<feature-group-spec name="access-control" xmlns="urn:jboss:galleon:feature-group:1.0">
+    <feature spec="core-service.management">
+        <feature spec="core-service.management.access.authorization">
+            <param name="provider" value="simple"/>
+            <feature spec="core-service.management.access.authorization.role-mapping">
+                <param name="role-mapping" value="SuperUser"/>
+                <feature spec="core-service.management.access.authorization.role-mapping.include">
+                    <param name="include" value="user-\$local"/>
+                    <param name="name" value="&quot;\$local&quot;"/>
```

Comment:
This escaping produces a different resource address than the current config does, one that I can't navigate to with the CLI. The following works and seems more correct too:
```
                    <param name="include" value="user-$local"/>
                    <param name="name" value="$local"/>
```